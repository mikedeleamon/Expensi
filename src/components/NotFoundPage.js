import React from "react";
import {Link} from 'react-router-dom'


const NotFoundPage = () => (
    <div>
        <h1>ERROR 404!</h1>
        <h3>PAGE NOT FOUND</h3>
        {/*use Link/NavLink to link internal pages*/}
        <Link to='/'>Back to Dashboard</Link>
    </div>
)

export default NotFoundPage