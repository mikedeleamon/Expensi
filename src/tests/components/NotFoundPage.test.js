import {shallow} from 'enzyme'
import React from 'react'
import NotFoundPage from '../../components/NotFoundPage'

test('should render error page', ()=>{
    const wrapper = shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()
})