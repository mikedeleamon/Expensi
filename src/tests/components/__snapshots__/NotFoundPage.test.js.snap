// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render error page 1`] = `
<div>
  <h1>
    ERROR 404!
  </h1>
  <h3>
    PAGE NOT FOUND
  </h3>
  <Link
    replace={false}
    to="/"
  >
    Back to Dashboard
  </Link>
</div>
`;
